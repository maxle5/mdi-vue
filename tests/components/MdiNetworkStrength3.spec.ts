
import { mount } from "@vue/test-utils";
import { expect, test } from "vitest";
import MdiNetworkStrength3 from "../../src/components/MdiNetworkStrength3.vue";

test("MdiNetworkStrength3 snapshot", () => {
  const wrapper = mount(MdiNetworkStrength3, {});
  expect(wrapper.html()).toMatchSnapshot();
});
