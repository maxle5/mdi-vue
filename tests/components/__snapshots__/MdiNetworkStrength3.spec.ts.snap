// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`MdiNetworkStrength3 snapshot 1`] = `
"<svg width="24" height="24" viewBox="0 0 24 24">
  <path d="M1,21H21V1M19,5.83V19H16V8.83" style="fill: currentColor;"></path>
</svg>"
`;
